packages/expo-notifications/src/scheduleNotificationAsync.ts

```diff
@@ -191,3 +186,86 @@ function isSecondsPropertyMisusedInCalendarTriggerInput(
       Object.keys(triggerWithoutChannelId).length > 1)
   );
 }
+
+function validateDateComponents(
```

Comment:
```suggestion
function validateDateComponentsInTrigger(
```

Reply:
oops! I just committed the suggestion but this won't work if I don't change where it's being called xD failing tests incoming! (I'll fix that soon)